eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -35,9 +35,9 @@ jobs:
     # Linux arm64
     - ${{ if eq(parameters.platform, 'Linux_arm64') }}:
       - ${{ if eq(parameters.jobParameters.isFullMatrix, true) }}:
-        - (Ubuntu.1804.ArmArch.Open)Ubuntu.1804.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm64v8-20210106155927-56c6673
+        - (Ubuntu.1804.ArmArch.Open)Ubuntu.1804.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm64v8
       - ${{ if eq(parameters.jobParameters.isFullMatrix, false) }}:
-        - (Ubuntu.1804.ArmArch.Open)Ubuntu.1804.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm64v8-20210106155927-56c6673
+        - (Ubuntu.1804.ArmArch.Open)Ubuntu.1804.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm64v8
```

Comment:
@sbomer , we were so close! Do you mind taking a look?

All the tests in one of the modified queues failed: The docker image was not found.

- Failed queue: `(Ubuntu.1804.ArmArch.Open)Ubuntu.1804.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm64v8`
- Job: `Libraries Test Run release mono Linux arm64 Debug`
- Test results: https://dev.azure.com/dnceng-public/public/_build/results?buildId=165041&view=logs&j=3f95ca23-483b-5dc9-4b05-6bd24fb925c2&t=0c1dd9d1-4ef7-54d5-2d62-e4ab330605ca
- Error message example:
```
Console log: 'Common.Tests' from job 548b59dc-5738-4816-bfb7-53d3d3295c8d (ubuntu.1804.armarch.open) using docker image mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm64v8 on a000SY7
running $HELIX_CORRELATION_PAYLOAD/scripts/e02d0a26b0e943fab7a86c4be842460d/execute.sh in /datadisks/disk1/work/A595092D/w/A06C08F8/e max 2700 seconds

Output:
Unable to pull image mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm64v8
Exit Code:-4
```

Reply:
@sbomer I notice this string is slightly different to the one that was used in the 7.0 PR (the hash at the end differs). If you don't mind, I'll use the one from the 7.0 PR, see if that fixes the failure.

Reply:
Oh I see the problem. We're attempting to use 16.04, that might be a typo.